### PR TITLE
draft output should go completely to stdout

### PR DESCRIPTION
**Pull request by obestwalter**
At least to me it does make no sense that the header goes to stderr and the rest to stdout.

### CODE REVIEW

File: src/towncrier/__init__.py [hunk 1]
```diff
@@ -103,8 +103,7 @@ def __main(draft, directory, project_version, project_date, answer_yes):
             "What is seen below is what would be written.\n",
             err=to_err,
         )
-        click.echo(top_line, err=to_err)
-        click.echo(rendered)
+        click.echo("%s%s" % (top_line, rendered))
```
Comment (michaeljoseph) [inline src/towncrier/__init__.py]: Looks like this breaks the tests since you dropped the newline.
Maybe just remove the `err` kwarg?

**Comment by obestwalter** [comment]
Thanks @michaeljoseph - added the missing newline.

**Comment by michaeljoseph** [comment]
The test failures look like tox 😤 travis miscommunications 👊 
So 👍 from me.

**Comment by hawkowl** [comment]
Travis removed pypy and python3.3. But this seems fine, thanks! The original idea was that the draft could be piped out, but I don't think many people would want to do that...

**Pull request merged**